ghcide/src/Development/IDE/GHC/Orphans.hs

```diff
@@ -132,24 +133,24 @@ srcSpanEndColTag = "srcSpanEndCol"
 instance ToJSON RealSrcSpan where
   toJSON spn =
       object
-        [ srcSpanFileTag .= unpackFS (srcSpanFile spn)
-        , srcSpanStartLineTag .= srcSpanStartLine spn
-        , srcSpanStartColTag .= srcSpanStartCol spn
-        , srcSpanEndLineTag .= srcSpanEndLine spn
-        , srcSpanEndColTag .= srcSpanEndCol spn
+        [ toJsonKey srcSpanFileTag .= unpackFS (srcSpanFile spn)
```

Comment:
In these cases I think you could exploit the fact that `Key` has a `IsString` instance (and so does `Text`):
1. Define `srcSpanFileTag` and friends as type `String`
2. Use `fromString srcSpanFileTag` here, which will work to produce both a `Key` and a `Text`.

Reply:
Thanks for the suggestion, not a big fan of `IsString`, but i dislike cpp even more :-)